docs/ROOM3_MIGRATION_GUIDE.md

```
@@ -0,0 +1,457 @@
+# Room 2 to Room 3 Migration Guide (Consumer Projects)
+
+**Applies to**: `mifos-pay`, `mifos-mobile`, `mifos-x-field-officer-app`, and any project consuming `kmp-project-template` via `sync-dirs.sh`
+
+**Last Updated**: 2026-04-30
+
+> **IMPORTANT: Kotlin Version Requirement**
+> Room 3.0-alpha03 klibs are compiled with Kotlin 2.3.20. You MUST upgrade Kotlin, KSP, and Compose Multiplatform:
+> | Dependency | Required Version |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)